ReactiveCocoa/Swift/Atomic.swift

```diff
@@ -25,25 +72,17 @@ public final class Atomic<Value> {
 	}
 	
 	/// Initializes the variable with the given initial value.
-	public init(_ value: Value) {
+	public init(_ value: Value, mutex: MutexType = PosixThreadMutex()) {
```

Comment:
I think it makes more sense to have `Atomic` parameterised with a `MutexType`. It also allows the compiler to specialise the calls to the `MutexType` implementation.